_sass/_objects--technologies-list.scss

```diff
@@ -0,0 +1,16 @@
+$size--technology-small:                         100% / 3;
+
+.technologies {
+  justify-content: space-around;
+
+  @include media-breakpoint-down(sm) {
```

Comment:
@TheMightyPenguin mobile first, use always `breakpoint-up` thus your mobile styles are first on the rule and them you apply specific styles for above resolutions